cli/src/utils.ts

```diff
@@ -106,6 +107,11 @@ export interface CrawlOptions {
   exclusionPattern?: string;
   extensions: string[];
 }
+
+const convertPathToPatternOnWin = (path: string) => {
+  return platform() === 'win32' ? convertPathToPattern(path) : path;
+};
```

Comment:
Does this need to be conditional to the platform, would there be any issues with just always converting?

Reply:
Yes, `convertPathToPattern` will change the matching behavior on *nix platform:

[`/photo*` will become `/photo\*`](https://www.npmjs.com/package/fast-glob#convertpathtopatternpath) thus the asterisk will lose its effect.

Reply:
But it doesn't change the behaviour on windows? If so that sounds weird, do you know why it is like that?

Reply:
No, it doesn't change the behaviour on Windows.

From my understanding, `convertPathToPattern` is a function especially used for converting a Windows path to `fast-glob` search pattern. It is meaningless to use this function if your inputs are a valid POSIX path.